docs/dxp/7.x/en/site-building/site-appearance/developer-guide/themes/introduction-to-themes.md

```diff
@@ -0,0 +1,50 @@
+# Introduction to Themes
+
+## What is a Theme?
+
+* What is a theme? Is it needed? What can you do with it?
+* Maybe link to the Getting Started article for changing a Site's theme to ground discussion
+
+Maybe reference the "theme anatomy", and if possible show a picture (otherwise link to an article in the same vein)
+
+## Possible Section on Just Using Themes in general in Liferay UI?
+
+[Screenshot of page with one of the non-classic themes?]
+
+Not sure if there's enough content past, how to change a theme (with link), for a whole section... but if so, put that here
+
+## Introduction to Clay and CSS Development?
+
+Not sure if it's needed to go into full detail here, but at least a general overview of how they tie into theme development, and why that's important. Then link to respective articles for more information
```

Comment:
I see this useful in the context of "Developing Themes", kind of a small subsection there, as a sibling of some others related to "Developing themes" (see below)